src/hiveboard_bridge/src/main.cpp

```diff
@@ -27,8 +32,23 @@ int getTcpServerPort() {
     return port;
 }
 
+std::string getRobotName() {
+    std::string robotName;
+
+    if (!ros::param::get("~ROBOT_NAME", robotName)) {
+        ROS_INFO("No ROBOT_NAME parameter was given. Using default value %s", DEFAULT_ROBOT_NAME);
+        robotName = DEFAULT_ROBOT_NAME;
+    }
+
+    return robotName;
+}
```

Comment:
Could we move those functions? or we will we do a cleanup of the main eventually?